libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -340,6 +345,9 @@ class AP_Motors {
     bool                _thrust_balanced;       // true when output thrust is well balanced
     float               _thrust_boost_ratio;    // choice between highest and second highest motor output for output mixing (0 ~ 1). Zero is normal operation
 
+    // motor options
+    AP_Int8             _options;
```

Comment:
I wonder if we should go right to uint16, we often find we fill up these options quickly and regret only having 8 bits. Having said there is not so much scope for options down here in motors as there is up in the vehicles.

Reply:
Fixed thanks